docs/integrations/huggingface.rst

```diff
@@ -0,0 +1,16 @@
+HuggingFace
+-----------
+
+The Hugging Face Hub has a great amount of pre-trained models and datasets available.
+
+It is easy to convert a Huggingface dataset to Lance dataset:
```

Comment:
Lance <3 HuggingFace

The HuggingFace Hub has become the go to place for ML practitioners to find pre-trained models and useful datasets. HuggingFace datasets can be written directly into Lance format by using the `lance.write_dataset` method. You can write the entire dataset or a particular split. For example: